NuRadioMC/SignalGen/emitter.py

```diff
@@ -130,8 +130,7 @@ def get_time_trace(amplitude, N, dt, model, full_output=False, **kwargs):
         trace[1, N // 2] = (1.0 - kwargs.get("polarization", 0.5)) ** 0.5 * amplitude
         trace[2, N // 2] = kwargs.get("polarization", 0.5) ** 0.5 * amplitude
     elif(model == "efield_idl1_spice"):
-        launch_zenith, _ = hp.cartesian_to_spherical(*kwargs["launch_vector"])
-        iN = None
+        launch_zenith, _ = np.rad2deg(hp.cartesian_to_spherical(*kwargs["launch_vector"]))
```

Comment:
this seems like a bug. The default units should be used always. I you really need to convert to degree, didive by `units.deg`. The `launch_angle` is already converted to `degree` when the file is opened.

Reply:
fixed